.github/workflows/main.yml

```diff
@@ -0,0 +1,103 @@
+name: Application Tests
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  linting:
+    name: Run Linting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+      - name: Download Cached Gems
+        uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: bundler-cache-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            bundler-cache-
+      - name: Install Dependencies
+        run: bundle install
+      - name: Run Brakeman
+        run: bundle exec brakeman
+      - name: Run Rubocop
+        run: bundle exec rubocop
+  build:
+    name: Test Application
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rubygems: [3.0.3, latest]
+    services:
```

Comment:
I think I have pointed out already somewhere, but I can't find my comment. Is there any reason to not use `docker-compose` to setup services since it is already present in project?